File: twoToOne.js
function longest(s1, s2) {
  let str = s1 + s2;
  let sortedString = str.split('').sort();
  const distinctValues = [...new Set(sortedString)];
  return distinctValues.join('');
}

const longest = (s1, s2) => {
 return [...new Set(s1 + s2)].sort().join('');
}

function longest(s1, s2) {
  return [...new Set([...s1.split(''), ...s2.split('')])].sort().join('');
}

longest("aretheyhere", "yestheyarehere");

File: two_to_one.rb
def longest(a1, a2)
  str = a1 + a2
  str.split('').uniq.sort.join('')
end

def longest(a1, a2)
  (a1 + a2).split('').uniq.sort.join
end

def longest(a1, a2)
  (a1 + a2).chars.uniq.sort.join
end

def longest(a1, a2)
  (a1 + a2).chars.sort.join.squeeze
end

longest("aretheyhere", "yestheyarehere")
